### ISSUE TITLE
metamorphic: increase likelihood of interesting SeekPrefixGEs

### ISSUE THREAD

**Issue by jbowens** [cockroachdb/pebble#1885]
The metamorphic tests don't have great coverage of SeekPrefixGE: The vast majority of SeekPrefixGE calls are either outside the iterator bounds or for prefixes that contain zero live keys.

We should improve the probability of generating SeekPrefixGE keys within the iterator bounds, and maybe give higher weight to keys that have been written before.

**Comment by bananabrick** [comment]
Flushing more context from slack:

```
cockroach uses SeekPrefixGE for gets, but our test coverage in the metamorphic tests is really poor. the majority of SeekPrefixGE ops it generates are just errors due to the seek key being outside the iterator bounds. even when the seek key is within the bounds, the SeekPrefixGE rarely finds any keys because we pull the seek key from the list of all keys we've ever generated (including keys that have never been written)
```

**Comment by jbowens** [comment]
Resolved by #1922.
